src/core/components/courses/SlotContainer.vue

```diff
@@ -338,6 +334,12 @@ export default {
 
       return 'to-plan';
     },
+    setIsOnlySlot (step) {
+      const days = Object.keys(this.courseSlotsByStepAndDate[step]);
+      if (days.length === 1 && this.courseSlotsByStepAndDate[step][days[0]].length === 1) return true;
+
+      return false;
```

Comment:
tu peux directement mettre `return days.length === 1 && this.courseSlotsByStepAndDate[step][days[0]].length === 1` non ?

Reply:
done